wolfssl/ssl.h

```diff
@@ -1065,6 +1065,8 @@ WOLFSSL_API int wolfSSL_make_eap_keys(WOLFSSL*, void* key, unsigned int len,
         #ifdef __PPU
             #include <sys/types.h>
             #include <sys/socket.h>
+        #elif defined(WOLFSSL_RIOT_OS)
+            #include <sys/_types/_iovec_t.h>
```

Comment:
This is a Linux/Mac include and doesn't appear to be portable. Have you considered using NO_WRITEV in settings.h for WOLFSSL_RIOT_OS?